### PR TITLE
chore(locale): change some typo ko-KR

### PR DESCRIPTION
**Pull request by ankhgerel**
# Description
I changed the words that were not appropriate words that were not appropriate

## Type of change
- Translation update

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I ran ESLint and other linters for modified files
- [x] I have performed a self-review of my own code and tested it
- [ ] I have commented my code, particularly in hard-to-understand areas
  (including JSDoc for methods)
- [x] My changes generate no new warnings
- [ ] My code needed automated testing. I have added them (this is optional task)

## Screenshots (if any)

Please do not use any external image service. Instead, just paste in or drag and drop the image here, and it will be uploaded automatically.

### CODE REVIEW

File: src/languages/ko-KR.js [hunk 1]
```diff
@@ -230,8 +230,8 @@ export default {
     "Channel Name": "채널 이름",
     "Uptime Kuma URL": "Uptime Kuma URL",
     aboutWebhooks: "Webhook에 대한 설명: {0}",
-    aboutChannelName: "Webhook 채널을 우회하려면 {0} 채널 이름칸에 채널 이름을 입력해주세요. 예: #기타-채널",
-    aboutKumaURL: "Uptime Kuma URL칸을 공백으로 두면 기본적으로 Project Github 페이지로 설정해요.",
+    aboutChannelName: "Webhook 채널을 무시하려면 {0} 채널 이름칸에 채널 이름을 입력해주세요. 예: #기타-채널",
+    aboutKumaURL: "Uptime Kuma URL칸을 공백으로 두면 기본적으로 Github Project  페이지로 설정해요.",
```
Comment (pemassi) [inline src/languages/ko-KR.js:234]:
```suggestion
    aboutKumaURL: "Uptime Kuma URL칸을 공백으로 두면 기본적으로 Github Project 페이지로 설정해요.",
```

Edit multiple space.

File: src/languages/ko-KR.js [hunk 2]
```diff
@@ -254,7 +254,7 @@ export default {
     octopushTypePremium: "프리미엄 (빠름) - 알림 기능에 적합해요)",
     octopushTypeLowCost: "저렴한 요금 (느림) - 가끔 차단될 수 있어요)",
     "Check octopush prices": "{0}에서 Octopush 가격을 확인할 수 있어요.",
-    octopushPhoneNumber: "휴대전화 번호 (intl format, eg : +33612345678) ",
+    octopushPhoneNumber: "휴대전화 번호 (intl format, 예시: +82 1023456789) ",
```
Comment (pemassi) [inline src/languages/ko-KR.js:257]:
```suggestion
    octopushPhoneNumber: "휴대전화 번호 (intl format, 예시: +821023456789) ",
```

Doesn't library require non-space input?

File: src/languages/ko-KR.js [hunk 3]
```diff
@@ -361,8 +361,8 @@ export default {
     topicExplanation: "모니터링할 MQTT Topic",
     successMessage: "성공 메시지",
     successMessageExplanation: "성공으로 간주되는 MQTT 메시지",
-    error: "error",
-    critical: "critical",
+    error: "오류",
+    critical: "크리티컬",
```
Comment (pemassi) [inline src/languages/ko-KR.js:365]:
```suggestion
    critical: "치명적인",
```
I guess this wording is better. How about you?

File: src/languages/ko-KR.js [hunk 4]
```diff
@@ -518,14 +518,14 @@ export default {
     "Show update if available": "사용 가능한 경우에 업데이트 표시",
     "Also check beta release": "베타 릴리즈 확인",
     "Using a Reverse Proxy?": "리버스 프록시를 사용하시나요?",
-    "Check how to config it for WebSocket": "웹소켓에 대한 설정 방법 확인",
+    "Check how to config it for WebSocket": "웹소켓 대한 설정 방법",
     "Steam Game Server": "스팀 게임 서버",
     "Most likely causes:": "원인:",
-    "The resource is no longer available.": "더이상 사용할 수 없어요.",
+    "The resource is no longer available.": "더이상 사용할 수 없어요...",
```
Comment (pemassi) [inline src/languages/ko-KR.js:524]:
```suggestion
    "The resource is no longer available.": "더 이상 사용할 수 없어요...",
```

Check the below link.

https://www.korean.go.kr/front/onlineQna/onlineQnaView.do?mn_id=216&qna_seq=120413

File: src/languages/ko-KR.js [hunk 5]
```diff
@@ -518,14 +518,14 @@ export default {
...
     "There might be a typing error in the address.": "주소에 오탈자가 있을 수 있어요.",
     "What you can try:": "해결 방법:",
     "Retype the address.": "주소 다시 입력하기",
     "Go back to the previous page.": "이전 페이지로 돌아가기",
-    "Coming Soon": "Coming Soon",
+    "Coming Soon": "Coming Soon...",
```
Comment (pemassi) [inline src/languages/ko-KR.js:529]:
```suggestion
    "Coming Soon": "곧 만나보세요.",
```

If we try to use Korean in all wording, I think this could be used.

**Review by pemassi — CHANGES_REQUESTED** [review]
I suggested few improvements, could you check the comments?

File: src/languages/ko-KR.js (hunk 3 above)
Comment (ankhgerel) [reply src/languages/ko-KR.js:365]: I don't think it's good because it's too translated.

File: src/languages/ko-KR.js (hunk 5 above)
Comment (ankhgerel) [reply src/languages/ko-KR.js:529]: I don't think it's good because it's too translated.

File: src/languages/ko-KR.js (hunk 2 above)
Comment (ankhgerel) [reply src/languages/ko-KR.js:257]: I tried to make it look better by using spacing.

Comment (pemassi) [reply src/languages/ko-KR.js:257]: It could lead a user to enter input as wrong format. I have checked the other languages and they choose non-space.

**Review by pemassi — APPROVED** [review]
LGTM!

**Pull request merged**